### PR TITLE
GCI: Enhance Model Zoo UI

### PR DESCRIPTION
**Pull request by ZeroZeroJedenJeden**
Redesigned and rewritten Zoo UI.

### CODE REVIEW

File: ide/static/css/zoo.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+#wrapper {
+    padding-left: 250px;
+}
+
+#sidebar-wrapper {
+    z-index: 1000;
+    position: fixed;
+    left: 200px;
+	top: 0px;
```
Comment (Ram81) [inline ide/static/css/zoo.css]: Can you fix the indentation

File: ide/static/css/zoo.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    -moz-transition: all 0.5s ease;
+    -o-transition: all 0.5s ease;
+    transition: all 0.5s ease;
+}
+
+
+#page-content-wrapper {
+    width: 100%;
+    padding: 20px;
+	padding-left: 200px;
```
Comment (Ram81) [inline ide/static/css/zoo.css]: Same here

File: ide/static/js/modelZoo.js [hunk 3]
```diff
@@ -2,80 +2,175 @@ import React from 'react';
...
+      this.refs.vqa.className = "hide";
+      if(id == "all") {
+        this.refs.recognition.className = " ";
+        this.refs.detection.className = " ";
+        this.refs.retrieval.className = " ";
+        this.refs.seq2seq.className = " ";
+        this.refs.caption.className = " ";
+        this.refs.segmentation.className = " ";
+        this.refs.vqa.className = " ";
+      } else if(id == "recognition")
```
Comment (Ram81) [inline ide/static/js/modelZoo.js]: Put all `else if` statements in new line

File: ide/static/js/modelZoo.js [hunk 4]
```diff
@@ -2,80 +2,175 @@ import React from 'react';
...
+    }
+    mouseClick(id) {
+      this.refs.recognition.className = "hide";
+      this.refs.detection.className = "hide";
+      this.refs.retrieval.className = "hide";
+      this.refs.seq2seq.className = "hide";
+      this.refs.caption.className = "hide";
+      this.refs.segmentation.className = "hide";
+      this.refs.vqa.className = "hide";
+      if(id == "all") {
```
Comment (Ram81) [inline ide/static/js/modelZoo.js]: missing space after `if` and before `(`

**Comment by Ram81** [comment]
@ZeroZeroJedenJeden Nice work, one suggestion for any selected category on model zoo sidebar can you highlight that. It is difficult to identify on which section I am.

**Comment by ZeroZeroJedenJeden** [comment]
- fixed indentation

- added model's name wrapping for small screen

- added current category indicator

**Comment by ZeroZeroJedenJeden** [comment]
- Added background to current category indicator.

**Pull request merged**